Miscellaneous-Server.md

```diff
@@ -292,23 +293,26 @@ WantedBy=multi-user.target
 ```
 
 ```ini
-#topaz_game.service
+#lsb_map.service
```

Comment:
While you're here, you may as well make all of this project name agnostic: xi_map.service, etc.

Reply:
I left lsb in the lines that read `/path/to/lsb`. Otherwise I replaced 'lsb' with 'xi'